safe-global/safe-wallet-web
# refactor: Remove safe token paused logic

## What it solves

Removes the logic that checks if the safe token is paused since its unpaused now.

## How to test it

1. Create a transaction and select the safe token
2. Observe no warning

## Checklist
